src/main/java/com/wanted/babdoduk/restaurant/domain/review/contoller/RestaurantReviewController.java

```diff
@@ -0,0 +1,72 @@
+package com.wanted.babdoduk.restaurant.domain.review.contoller;
+
+import com.wanted.babdoduk.common.response.ApiResponse;
+import com.wanted.babdoduk.restaurant.domain.review.dto.RestaurantReviewRequestDto;
+import com.wanted.babdoduk.restaurant.domain.review.dto.RestaurantReviewResponseDto;
+import com.wanted.babdoduk.restaurant.domain.review.service.ReviewService;
+import com.wanted.babdoduk.restaurant.domain.review.service.ReviewStatService;
+import jakarta.validation.Valid;
+import java.net.URI;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/restaurants/{restaurantId}/reviews")
+public class RestaurantReviewController {
+
+    private final ReviewService reviewService;
+    private final ReviewStatService reviewStatService;
+
+    @PostMapping()
```

Comment:
```suggestion
    @PostMapping
```
사용하지 않는 `()` 는 제거하셔도 될 것 같습니다 😄

Reply:
말씀하신대로 수정했습니다!